test/chartist.component.spec.ts

```diff
@@ -156,24 +149,22 @@ describe('chartist component', function(): void {
     });
   });
 
-  it('should throw an error when missing type', function(): void {
+  it('should throw an error when missing type', function (): void {
     instance.data = data['Bar'];
 
     expect(instance.ngOnInit).toThrow();
   });
 
-  it('should throw an error when missing data', function(): void {
+  it('should throw an error when missing data', function (): void {
     instance.type = 'Bar';
 
     expect(instance.ngOnInit).toThrow();
   });
 
-  it('should throw an error when an invalid chart type is passed', function() {
+  it('should throw an error when an invalid chart type is passed', function (): void {
     instance.data = data['Bar'];
-    instance.type = 'NotAChart';
-
     instance.renderChart().catch((err) => {
-      expect(err.message).toBe('NotAChart is not a valid chart type');
+      expect(err.message).toBe('undefined is not a valid chart type');
```

Comment:
Hello willsoto, 

I'm sorry, I have activated auto format on commit. Yes, I have got some typescript/tslint errors and a warning by IntelliJ regarding let and const instead of var. But some of this changes seem to be quite necessary as far as I can see. But take a look by yourself, please:

```
**Using:**

C:\projects_git\ng-chartist>node -v
v7.10.0
C:\projects_git\ng-chartist>npm -v
4.2.0

typescript@2.3.3
C:\projects_git\ng-chartist\tsconfig.json

tslint@5.2.0
C:\projects_git\ng-chartist\tslint.json

**Running test:**

C:\projects_git\ng-chartist>npm test

> ng-chartist@1.2.0 test C:\projects_git\ng-chartist
> karma start


START:
Warning: The 'no-use-before-declare' rule requires type checking

[at-loader] Using typescript@2.3.3 from typescript and "tsconfig.json" from C:\projects_git\ng-chartist/tsconfig.json.


[at-loader] Checking started in a separate process...

[at-loader] Checking finished with 1 errors
Hash: 039ba0472d9da447106a
Version: webpack 2.6.1
Time: 27423ms
                Asset    Size  Chunks                    Chunk Names
config/spec-bundle.js  8.3 MB       0  [emitted]  [big]  config/spec-bundle.js

WARNING in ./test/chartist.component.spec.ts
[1, 8]: You need to leave whitespaces inside of the import statement's curly braces
[5, 8]: You need to leave whitespaces inside of the import statement's curly braces
[1, 48]: " should be '
[3, 27]: " should be '
[5, 44]: " should be '
....
```

This "fixed" test was also due to a ts error (TS2322):
![ts2322](https://cloud.githubusercontent.com/assets/11350261/26495418/c5e6ae4c-4223-11e7-8fab-69eeb36b782c.png)

The reason for my enhancement in tsconfig.json (commit id: 9c089b3):
![ts1219](https://cloud.githubusercontent.com/assets/11350261/26495556/5a2fc552-4224-11e7-9346-8e103cb10552.png)

This is the warning:
![var_used_instead_of_let_or_const](https://cloud.githubusercontent.com/assets/11350261/26495643/a8beb0a2-4224-11e7-9a87-e7060ff679db.png)

I'll revert my changes and do it in a more reviewer-friendly way, okay?!

Best regards,
Simon